go/tools/gopackagesdriver/aspect.bzl

```diff
@@ -52,16 +52,19 @@ def _go_archive_to_pkg(archive):
         ExportFile = _file_path(archive.data.export_file),
         GoFiles = [
             _file_path(src)
-            for src in archive.data.orig_srcs if not src.path.endswith(".s")
+            for src in archive.data.orig_srcs
```

Comment:
Could you remove the changes to this file from the PR (assuming they are the automated reformat I think the are)?

Reply:
Yeah my mistake, removed.